java/fury-core/src/main/java/org/apache/fury/ThreadLocalFury.java

```diff
@@ -266,8 +268,18 @@ public ClassLoader getClassLoader() {
   }
 
   @Override
-  public ClassResolver getClassResolver() {
-    return bindingThreadLocal.get().get().getClassResolver();
+  public void setClassChecker(ClassChecker classChecker) {
+    bindingThreadLocal.get().get().getClassResolver().setClassChecker(classChecker);
```

Comment:
We need to use processCallback

Reply:
> We need to use processCallback

I don't quite understand what you mean. Could you give me an example?

Reply:
You can take `setSerializerFactory` as an example:
```java
  public void setSerializerFactory(SerializerFactory serializerFactory) {
    processCallback(fury -> fury.setSerializerFactory(serializerFactory));
  }
```

Reply:
And maybe we can add following API for extending:
```java

public void configureClassResolver(Consumer<ClassResolver> apply);

```

Reply:
> You can take `setSerializerFactory` as an example:
> 
> ```java
>   public void setSerializerFactory(SerializerFactory serializerFactory) {
>     processCallback(fury -> fury.setSerializerFactory(serializerFactory));
>   }
> ```

I feel doing it this way might be simpler.